### PR TITLE
Footer/ WWD/ Contact Page: design review tweaks

### LINKED ISSUE #3374: Intro Hero Review
Pulling this out of #3367, since it affects every page except the home page, and I wanted to make sure the discussion isn’t lost. I'll refer to this component as "Intro Hero" because it seems like "Section Header" might be ambiguous. 

image

My original suggestion

- On the heading the background is `#162e51` which is not a brand color. Should be `#1C304A`
- The text is more “chunky” than our current site, despite being the exact same weight. I think this might be due to some “font-normalization” that USWDS does. We could compensate for that and  make it more readable by adding some letterspacing. 
Existing site:
image
Branch:
image

 
The other issue was about the size.
“Looking at other pages on the current site, I realized the H1 ought to be smaller. as well, as it's being used as a kind of pre-header text. Size should be ~2.3 rem, so something like lg should work?”

On our current site the name of the page, the `H1` is smaller than the “tagline” text below it. I believe the original design is trying to emphasize the description of the page content over the title. I think this works because the current page is also indicated in the navigation and the page name does not necessarily need to be overly emphasized. Because is only one `H1` on the page and it appears before the other headings, the `H1` makes sense semantically and in the logical page structure, and so I don’t believe that it causes any accessibility issues. 

Having said that, if we prefer it to be a big header, my main issue with the typography on the branch is that the tagline is only slightly different in size from the header. I’d like to differentiate it, in order to make clear that it’s not a continuation of the same text, and to make the difference seem intentional. So in that case I would propose we shift the weight to regular and try it at size `md`

### PR DESCRIPTION
**Pull request by lauraGgit**
**SEE the project board for which items are in progress/ done of the design label tickets.**

Incorporate HWW and netlify changes from existing site.

Close #3374

 ## Contact Page #3375 
**Intro Hero**
- [X] Changed based on #3374

**Link text**
- [X] Link color is `#046B99`

**Signup**
Direction A
- [X] Styled as in mockup, matching Call to Actions on other pages

  
## Footer: #3368
- [x] Headers are size `lg`
- [x] Headers are static text with no interaction effects. 
- [X] Links are `#046B99`

## What We deliver: #3376
**Intro text**
- [x] Rule is added back in, is `gray-cool-10 `(`#dfe1e2`)
- [x] 5 units of space after intro text and then 5 units of space after the `hr`  (above “Case Studies”)

To be discussed: 
- [x] text is `lg` and takes up 9 columns instead of 8

**Cards**
- [x] No border on the cards
- [x] Rule after cards, with 5 units of space between the cards and the line
- [x] Rule is is `gray-cool-10 `(`#dfe1e2`)

**Quote**
- [x] Style to new homepage designs.

**Agencies**
- [x] 5 units of space between heading and top of grey container
- [x] Agency list item has 2 units of space below

**Call to Action**
- [x] Background color should be `#1C304A`  
- [x] Text is lg size
- [x] Button has background color `#00CFFF` and font-color `#1C304A`.  (can follow the [ACCENT COOL COLOR ](https://designsystem.digital.gov/components/button/)example for states)

### CODE REVIEW

File: _includes/testimonial.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<section class="usa-section grid-container testimonial">
+    <div class="grid-row grid-gap">
+      <div class="tablet:grid-col-8 testimonial__body">
+        "{{ include.quote }}""
```
Comment (igorkorenfeld) [inline _includes/testimonial.html:4]: Extra quote mark here

**Review by igorkorenfeld — CHANGES_REQUESTED** [review]
## Contact
- [ ] Sign up area has a `</Section>` showing up on the right hand side of the rendered section 
image

- [x] “Sign up for the 18F newsletter” let’s bump up to `xl`
- [ ] Sidebar missing borders above the headers. The patterns should follow the other sidebar designs, so this might need to be a separate issue / PR. 
Current site:
image

## WWD
### Cards
- [x] Lost the rounded corners

Some stuff I missed the first time around (included it on home but not here): 
- [x] Add 0.5 units of space between the agency name and the project title
- [ ] Chevron next to read more should be smaller and aligned with the x-height of the text

### Quote
- [x] The characters at the start of and end the quote are “prime-marks” (`&#8243;`), which are straighter,  instead of quote marks	(`&#8220;`  and `&#8221; `), which have a little bit of a curve. There is also an extra one mark at the end of the quote. Can we change to  (`&#8220; ` and `&#8221;`  / `&ldquo;` `&rdquo;`). I am wondering if that extra mark is what's causing them to be converted to prime marks instead of quote marks
- [ ] The Treasury logo is coming in a little pixelated, can we use an [SVG file](https://upload.wikimedia.org/wikipedia/commons/c/cb/Seal_of_the_United_States_Department_of_the_Treasury.svg)


## Footer
Looks good!

**Review by peterrowland — APPROVED** [review]
√

**Comment by lauraGgit** [comment]
@igorkorenfeld and @peterrowland  thanks for the review.

@igorkorenfeld I made most of your changes, but the carets get fixed more globally in #3386 so holding off on those in this PR.

I also added the border to the sidenav but am holding off on more tweaks until a bit later to incorporate later changes. (too many prs!)

**Comment by igorkorenfeld** [comment]
Looks good to me!

**Review by igorkorenfeld — APPROVED** [review]

**Pull request merged**